packages/react-sdk/README.md

```diff
@@ -80,6 +80,8 @@ import { BucketProvider } from "@bucketco/react-sdk";
   <BucketProvider>
   ```
 
+- `impersonating` (default: `false`): Disables sending tracking events and user/company updates to Bucket and disabled automated feedback surveys. Set to `true` when impersonating another user.
```

Comment:
The wording should be "enables".

Reply:
OK i can see how this is confusing. If you set it to something other than the default then it'll disable sending tracking events .... Needs a rewording.

Reply:
OK let me know if this is better.